tests/client/amo/components/TestCategeories.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+import { Provider } from 'react-redux';
+
+import createStore from 'amo/store';
+import Categories, { filterAndSortCategories } from 'amo/components/Categories';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+const categories = [
+  {
+    application: 'firefox',
+    name: 'motorbikes',
+    slug: 'motorbikes',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Travel',
+    slug: 'travel',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Games',
+    slug: 'games',
```

Comment:
Should probably have some slugs that aren't just lowercase of the name to be safe.